metron-platform/metron-solr/src/main/java/org/apache/metron/solr/dao/SolrSearchDao.java

```diff
@@ -238,19 +204,19 @@ protected SearchResponse buildSearchResponse(
     return searchResponse;
   }
 
-  protected SearchResult getSearchResult(SolrDocument solrDocument, Optional<List<String>> fields) {
-    SearchResult searchResult = new SearchResult();
-    searchResult.setId((String) solrDocument.getFieldValue(Constants.GUID));
-    Map<String, Object> source;
-    if (fields.isPresent()) {
-      source = new HashMap<>();
-      fields.get().forEach(field -> source.put(field, solrDocument.getFieldValue(field)));
-    } else {
-      source = solrDocument.getFieldValueMap();
-    }
-    searchResult.setSource(source);
-    return searchResult;
-  }
+//  protected SearchResult getSearchResult(SolrDocument solrDocument, Optional<List<String>> fields) {
```

Comment:
Deleted 1 commented line, added 13 more? Haha.

Reply:
My bad.  The merge from the feature branch was messy and I didn't clean up after myself.